src_py/_audio.py

```
@@ -0,0 +1,432 @@
+            raise ValueError("Gain must be >= 0.")
+
+        _base_audio.set_audio_stream_gain(self._state, value)
+
+    @property
+    def frequency_ratio(self) -> float:
+        return _base_audio.get_audio_stream_frequency_ratio(self._state)
+
+    @frequency_ratio.setter
+    def frequency_ratio(self, value: float) -> None:
+        if value <= 0 or value > 10:
+            raise ValueError("Frequency ratio must be > 0 or <= 10.")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)